src/or/nodelist.c

```diff
@@ -891,6 +891,15 @@ node_get_by_hex_id(const char *hex_id, unsigned flags)
   return NULL;
 }
 
+const smartlist_t *
```

Comment:
Hmm why do we need to define a new function for this? We already have `nodelist_get_list()` that does this.
See how it's used in other parts of the code. :)

Reply:
Fixed.